src/logic/README.md

```diff
@@ -0,0 +1 @@
+- All reusable logic should be saved within this folder if its intended to be reused across multiple `vets-website` applications.
```

Comment:
Okay, good to know - I didn't know this, or didn't recall this was the case - I can definitely move these files there & will update the PR accordingly.

Reply:
After discussing with my team, we decided that it still makes the most sense for everything in `src/platform/utilities/legacy` to be moved to the `src/applications/proxy-rewrite` directory, since it's not being used anywhere else in the repo, and is specific to that application as of now. We also decided that `isBrowserIE` and `useOnLoaded` are better suited in the `src/platform/site-wide` directory. Since your team (Public Websites) is a codeowner of that directory, feel free to add those functions in there. 

Because Release Tools is the codeowner of the `src/platform` directory as a whole, we follow certain criteria for evaluating additions/changes to that directory, including:
- Single instance helper functions (used in one application) should not be added to `src/platform`, except for subdirectories that your team is a codeowner for, i.e. `src/platform/site-wide`
- If single instance helper functions are intended to be used elsewhere in future code, they can be reconsidered to be added to `src/platform` at that time

Once these changes are made, we can review your PR again.